app/controllers/spotlight/concerns/application_controller.rb

```diff
@@ -13,7 +13,11 @@ module ApplicationController
       helper Spotlight::ApplicationHelper
 
       rescue_from CanCan::AccessDenied do |exception|
-        redirect_to main_app.root_url, :alert => exception.message
+        if current_exhibit and !can? :read, current_exhibit
+          redirect_to main_app.new_user_session_url, :alert => exception.message
```

Comment:
Heh, I was kinda joking (hence the stern warning part).

Anyhow, this ends up being problematic if it is the default exhibit that is unpublished because a logged in user could get into the situation where they get to an error page and are never able to navigate back to their exhibit unless they know the URL by heart.

I understand that this is really an application concern, and they can decide what the correct behavior is for them in this case, but it may be something we should document down the road.